Buildings/Fluid/HeatExchangers/BaseClasses/lmtd.mo

```diff
@@ -27,9 +27,7 @@ revisions="<html>
 <ul>
 <li>
 November 30, 2020, by Michael Wetter:<br/>
-Updated documentation.<br/>
-This is for
-<a href=\"https://github.com/lbl-srg/modelica-buildings/issues/2268\">#2268</a>.
```

Comment:
It would be best to include them, but use a format like
```
<a href=\"https://github.com/lbl-srg/modelica-buildings/issues/2268\">Buildings, #2268</a>
```
so it is clear that the number is from another repository.

Reply:
Done.